.github/workflows/test.yml

```diff
@@ -0,0 +1,102 @@
+name: Build & Test
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+jobs:
+  job_test:
+    name: Test
+    runs-on: ubuntu-latest
```

Comment:
does it compile/test for iOS even being ubuntu? doesn't need it to be macos?

Reply:
Ah so the Bundle step only builds the JS code bundle and not building the Java/Obj-C code. I was considering adding that as a test to this too but I think it would be redundant with the e2e tests already testing that.